### PR TITLE
Snippets to backticks instead of quotes

### PR DESCRIPTION
**Pull request by knod**
Can't use the backticks for prop names, so they'll have to be in quotes if we're going to stick with the versioning system ~we currently have~ [we're currently planning].

### CODE REVIEW

**Comment by wacii** [comment]
These aren't the same. Newlines in template literals count as newlines.

```js
str1 = `a
b`;
str2 = "a\nb";
// str1 == str2
```

**Comment by knod** [comment]
@wacii : I thought I got rid of all of them. Which lines have I missed?

**Comment by ethanbb** [comment]
So the main purpose of this change is just to get rid of escaped quotes within the snippets, correct?

**Comment by knod** [comment]
@ethanbb : Yes, and to get rid of strings put together with '+'. There are a bunch of changes that need to be made to snippets, and this is the first, and probably most simple, step.

File: src/localization/de.js [hunk 1]
```diff
@@ -1,171 +1,135 @@
...
+    why2: `Die Cliff Effects sind schwer vorherzusehen. Die Interaktionen zwischen dem Einkommen, der Nummer von Personnen im Haushalt, vielen verschiedenen anderen Krieterien, und den Auswirkungen der Programmen selbst haben Auswirkungen aufeinander in unerwarteten Weisen. Wir suchen Methoden, um dieses Problem sich mit der Komplexität zu befassen, und die Familien ihre Situationen besser zu verstehen und vorherzusehen.`,
     
-    linkText1: 'Ein zweiminutenlanges Video über Cliff Effects',
-    linkText2: 'Manche Quantitative Szenarios, die Cliff Effects demonstrieren',
-    linkText3: 'Eine Aufgliederung von verschiedenen Sozialhilfeprogrammen im Bundesstaat ' +
-      'Massachusetts',
+    linkText1: `Ein zweiminutenlanges Video über Cliff Effects`,
+    linkText2: `Manche Quantitative Szenarios, die Cliff Effects demonstrieren`,
+    linkText3: `Eine Aufgliederung von verschiedenen Sozialhilfeprogrammen im Bundesstaat ` +
+      `Massachusetts`,
```
Comment (wacii) [inline src/localization/de.js]: These lines weren't combined.

**Comment by knod** [comment]
Yeah, that was part of the reasoning. The `+`s are very dependent on the text there and they make editing fragile.

**Comment by zhujinxuan** [comment]
I think this can be automatically fixed by prettier.  I will raise that issue in next week.  (Sorry I fail to attend this Tuesday)

**Pull request merged**